src/plugins/intel_gpu/src/kernel_selector/cl_kernels/quantize_gpu_scale_shift_vload8_opt.cl

```diff
@@ -0,0 +1,129 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "include/batch_headers/fetch_data.cl"
+#define INPUT0_VEC_TYPE  MAKE_VECTOR_TYPE(INPUT0_TYPE, 8)
+#define INPUT1_VEC_TYPE  MAKE_VECTOR_TYPE(INPUT1_TYPE, 8)
+#define OUTPUT_VEC_TYPE  MAKE_VECTOR_TYPE(OUTPUT_TYPE, 8)
+
+#define TO_VECTOR_TYPE_IMPL_8(elem_type)  CAT(convert_##elem_type, 8)
+#define TO_VECTOR_TYPE(elem_type, size)   CAT(TO_VECTOR_TYPE_IMPL_, size)(elem_type)
+#define TO_VECTOR_TYPE_IMPL_SAT_8(elem_type)  CAT(convert_##elem_type, 8##_sat)
+#define TO_VECTOR_TYPE_IMPL_SAT_RTE_8(elem_type)  CAT(convert_##elem_type, 8##_sat_rte)
+#define TO_VECTOR_TYPE_SAT(elem_type, size)   CAT(TO_VECTOR_TYPE_IMPL_SAT_, size)(elem_type)
+#define TO_VECTOR_TYPE_SAT_RTE(elem_type, size)   CAT(TO_VECTOR_TYPE_IMPL_SAT_RTE_, size)(elem_type)
+// Convert built-in functions with _sat modifier are not supported in floating point
+//create defines without _sat to overcome this issue
+#define convert_float8_sat  convert_float8
+#define convert_half8_sat   convert_half8
+
+#ifdef SUB_GROUP_SIZE
+REQD_SUB_GROUP_SIZE(SUB_GROUP_SIZE)
+#endif
+#ifndef IS_DYNAMIC
+__attribute__((reqd_work_group_size(LWS_0, LWS_1, LWS_2)))
+#endif
+
+KERNEL(quantize_gpu_scale_shift_vload8_opt)(OPTIONAL_SHAPE_INFO_ARG
+                                     const __global INPUT0_TYPE* input,
+                                     const __global INPUT1_TYPE* input_low,
+                                     const __global INPUT2_TYPE* input_high,
+                                     const __global INPUT3_TYPE* output_low,
+                                     const __global INPUT4_TYPE* output_high,
+                                     const __global INPUT5_TYPE* input_scale,
+                                     const __global INPUT6_TYPE* input_shift,
+                                     const __global INPUT7_TYPE* output_scale,
+                                     const __global INPUT8_TYPE* output_shift,
+                                           __global OUTPUT_TYPE* output)
+{
+    const int global_id = get_global_id(0);
+#if LAST_ACCESSED_X
+    global_id -= max(global_id - (int)LAST_ACCESSED_X, 0);
+#endif
+    const INPUT0_VEC_TYPE in0 = vload8(global_id, input);
```

Comment:
It seems leftovers handling is not finished yet?

Reply:
updated with leftover handling